src/app/to-do/to-do.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-to-do',
+  templateUrl: './to-do.component.html',
+  styleUrls: ['./to-do.component.css']
+})
+export class ToDoComponent {
+  public items: string[] = [];
+
+  public newTask: any;
+
+  public addToList() {
+    if (this.newTask == ''){
+    }
+    else{
+      this.items.push(this.newTask);
+      this.newTask = '';
+    }
+  }
+
+  public deleteTask(index: any) {
```

Comment:
instead of `any` this is probably `number`?

Reply:
This has been corrected in most recent commits.